dask_match/core.py

```diff
@@ -553,110 +553,10 @@ class IO(API):
     pass
 
 
-class ReadParquet(IO):
-    """
-
-    This isn't really built out yet.  We only have metadata for this but no
-    actual reading of real parquet data.  It's useful today mostly as a prop
-    for optimization.
-    """
-
-    _parameters = ["filename", "columns", "filters"]
-    _defaults = {"columns": None, "filters": None}
-
-    @staticmethod
-    def normalize(filename=None, columns=None, filters=None):
-        if isinstance(columns, tuple):
-            columns = list(columns)
-        return (
-            filename,
-            columns,
-            filters,
-        ), {}
-
-    @property
-    def _meta(self):
-        # This is complete crap, but again, useful for optimization practice
-        df = pd.DataFrame({"a": [1], "b": [2.0], "c": [4], "d": [5.0]})
-        if self.columns is not None:
-            df = df[self.columns]
-        return df.head(0)
-
-    @classmethod
-    def _replacement_rules(cls):
-        _ = Wildcard.dot()
-        a, b, c, d, e, f = map(Wildcard.dot, "abcdef")
+def ReadParquet(*args, **kwargs):
+    from dask_match.io.parquet import ReadParquet
```

Comment:
If we're going with a normal function then let's call it `read_parquet` as dask.dataframe does.  I only used CamelCase previously to respect Python class conventions.

Reply:
Totally agree - Just a temporary redirect to avoid changing some of the existing tests while I played around.